app/policies/appliance_comment_policy.rb

```diff
@@ -0,0 +1,7 @@
+class ApplianceCommentPolicy < ApplicationPolicy
+
```

Comment:
I think we sholud not add new line befora and after this function. We expressed the logical separation with indentation.